### PR TITLE
[DinoMod] elephant armor

### PR DESCRIPTION
**Pull request by LyleSY**
#### Summary

SUMMARY: Mods "Elephant armor that fits some dinos"

#### Purpose of change

Allows players to craft armor for elephants and dinosaurs that are built like elephants such as ceratopsians

#### Describe the solution

Reskin of horse armor with elephant and Indian culture and history references, one dino variant. Also bugfix for horse kevlar armor recipe accidentally copied and caught here. Good catch @Coolthulhu 

#### Describe alternatives you've considered

Ostrich version, may still do that

#### Testing

Game loads no errors, items spawn, look and work as expected

Screen Shot 2021-05-31 at 3 28 51 PM

Screen Shot 2021-05-31 at 3 30 22 PM

#### Additional context

"I will either find a way, or make one." -Hannibal

### CODE REVIEW

**Comment by LyleSY** [comment]
xcode errors don't look related

File: data/mods/DinoMod/recipes/pets_elephant.json [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+    "result": "kevlar_armor_elephant",
+    "type": "recipe",
+    "category": "CC_ANIMALS",
+    "subcategory": "CSC_ANIMALS_ELEPHANT ARMOR",
+    "skill_used": "tailor",
+    "difficulty": 6,
+    "time": "400 m",
+    "autolearn": true,
+    "using": [ [ "sewing_standard", 400 ] ],
+    "components": [ [ [ "kevlar_plate", 100 ] ], [ [ "rag", 100 ] ], [ [ "leather", 50 ], [ "tanned_hide", 50 ] ] ]
```
Comment (Coolthulhu) [inline data/mods/DinoMod/recipes/pets_elephant.json:50]: 50 tanned hide is wrong: one tanned hide is 6 leather pieces.

**Comment by Coolthulhu** [comment]
The armors could use a category in crafting menu. Currently they're only visible under "all".

File: data/mods/DinoMod/recipes/pets_elephant.json (hunk 1 above)
Comment (LyleSY) [reply data/mods/DinoMod/recipes/pets_elephant.json:61]: Ha, that error was copied from the horse original. Good eye!

**Pull request merged**